zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -344,6 +346,24 @@ public int progress() {
   }
 
   public boolean isBlankParagraph() {
```

Comment:
rename it to shouldSkipRunParagraph. Because what this function doing is not actually checking whether it is blank after this PR.

Reply:
ok.
Also `checkEmpty = false`, Is there a more appropriate name for `checkEmpty`?